src/NzbDrone.Core/Download/ProcessMonitoredDownloadsCommand.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace NzbDrone.Core.Download
 {
-    public class CheckForFinishedDownloadCommand : Command
+    public class ProcessMonitoredDownloadsCommand : Command
```

Comment:
This breaks the api.

Reply:
In what sense? The rename is for clarity, which makes sense I think.

Reply:
Coz third parties can publish the command via the api. Granted it's not very likely in this particular case, but I think we should maintain backward compat.

Reply:
Re-added the old command and will log a deprecated message (warning) is someone actually calls it.